cli/pcluster/config_sanity.py

```diff
@@ -291,6 +297,7 @@ def validate(self, resource_type, resource_value):  # noqa: C901 FIXME
                     boto3.client(
                         "sts",
                         region_name=self.region,
+                        endpoint_url=self.sts_endpoint_url,
```

Comment:
Then, if the sts_endpoint_url is only used here we can avoid to define an object attribute and just call here the `__get_sts_endpoint_url()` function